programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -500,6 +501,23 @@ static bool process_tunnel_commands(const tunnel_command *tnl_cmd, command_cb cb
             free_tunnel_service_control(&tunnel_service_control_opts);
             break;
         }
+        case TunnelCommand_StatusChange: {
```

Comment:
added this tunnel status command, so we can make this function available from command line. That will help in testing this command without laptop resume operation.